addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/dhcp/DHCPPacket.java

```diff
@@ -88,6 +88,18 @@
     private Map<Byte, byte[]> options;
 
     /**
+     * Package private constructor for test suite.
```

Comment:
Good that you add the comment but it would be great if it can be avoided for example by having a testbuilder in the test package.